src/Neo.CLI/config.fs.testnet.json

```diff
@@ -36,6 +36,11 @@
     "MaxTraceableBlocks": 2102400,
     "InitialGasDistribution": 5200000000000000,
     "ValidatorsCount": 7,
+    "MemoryPool": {
+      "EnableSmartThrottler": true,
+      "MaxTransactionsPerSecond": 512,
+      "MaxTransactionsPerSender": 100
```

Comment:
This does not look good,Jimmy.
Limit sender does not make much sense in Blockchain to me

Reply:
Has removed the limitation to sender